spec/controllers/nonprofits/tag_joins_spec.rb

```diff
@@ -11,6 +11,53 @@
 
     describe 'modify' do
       include_context :open_to_np_associate, :post, :modify, nonprofit_id: :__our_np, id: '1'
+
+      context 'strong params' do
```

Comment:
This context is a bit misleading at the moment, although I'm not sure we need to change it...

Reply:
In this case, it's not really misleading, that one is being handled by Strong Params.